docs/docs-components/siteIndex.js

```diff
@@ -175,6 +175,7 @@ const siteIndex: $ReadOnlyArray<siteIndexType> = [
         sectionName: 'Color',
         pages: ['Palette', 'Usage', 'Examples'],
       },
+      'Content Standards',
```

Comment:
We should check with Brie, but I think this should be sentence-case.

Reply:
+1, this should be "Content standards"